pwnlib/util/crc/__init__.py

```diff
@@ -28,6 +10,27 @@
 from pwnlib.util import safeeval
 from pwnlib.util.crc import known
 
+class Nothing:
```

Comment:
Why create a class called `Nothing` and move the module docstring into `Nothing` docstring?

Reply:
It is a dirty hack. The oldest commit does not pass the test. I wanted to fix it before continue to write my own PR.

Reply:
The `fix das build` commit break the changes : https://github.com/Gallopsled/pwntools/commits/dev